gstd/gstd.c

```diff
@@ -63,86 +56,16 @@ int_term_handler (gpointer user_data)
   /* User has pressed CTRL-C, stop the main loop
      so the application closes itself */
   GST_INFO ("Interrupt received, shutting down...");
+  g_print ("\n");
```

Comment:
Why this is required?, should it part of GST_INFO (GST_INFO ("Interrupt received, shutting down...\n");)

Reply:
It's just to have an output like: 
<pre><code>^C
jmontoya@jmontoya-work:~$ 
</code></pre>

Insted of:
<pre><code>^Cjmontoya@jmontoya-work:~$ 
</code></pre>

And it is not at the end of GST_INFO because that's not a print on screen but in a file